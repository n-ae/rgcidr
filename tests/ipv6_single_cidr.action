2001:db8::1/128
