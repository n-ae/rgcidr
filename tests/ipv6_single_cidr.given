2001:db8::1
2001:db8::2
2001:db8::
2001:db8:0000:0000:0000:0000:0000:0001
2001:db8:0:0:0:0:0:1
2001:db9::1
fe80::1
