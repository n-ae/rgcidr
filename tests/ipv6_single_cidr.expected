2001:db8::1
2001:db8:0000:0000:0000:0000:0000:0001
2001:db8:0:0:0:0:0:1
